var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -234,6 +240,12 @@ def configure_args(self):
             if "+external-xdr" in hdf4 and hdf4["rpc"].name != "libc":
                 libs.append(hdf4["rpc"].libs.link_flags)
 
+        if "+zstd" in self.spec:
+            zstd = self.spec["zstd"]
+            config_args.append("--with-plugin-dir={}".format(self.prefix.plugins))
+            cflags.append("-I" + zstd.prefix.include)
+            ldflags.append("-L" + zstd.prefix.lib)
```

Comment:
1. `-I` should go to `cppflags`
2. Is it possible to use `headers` and `libs`? For example:
    ```python
    cppflags.append(zstd.headers.cpp_flags)
    ldflags.append(zstd.libs.search_flags)
    libs.append(zstd.libs.link_flags)
    ```

Reply:
Yes, but zstd does not currently define those properties.

Reply:
Or does it automatically work?

Reply:
There are default implementations for the properties. If they work correctly, please, use them. Otherwise, leave a `TODO` comment as for other such cases.

Reply:
You can drop `libs.append(zstd.libs.link_flags)` though, the build system tests for `-lzstd`.

Reply:
The default implementations work